src/app.zig

```diff
@@ -826,15 +826,17 @@ pub const App = struct {
     }
 
     /// handle a command
-    pub fn handleCommand(self: *App, lua_state: *Lua, buffer: irc.Buffer, cmd: []const u8) !void {
+    pub fn handleCommand(self: *App, lua_state: ?*Lua, buffer: irc.Buffer, cmd: []const u8) !void {
```

Comment:
I don't understand why you needed to make `lua_state` an optional here. In the next commit when you call this function you pass a `null`, but you could just as easily pass the state, right?

Reply:
It wasn't necessary, I just wasn't sure whether you would want lua state to be passed around into the draw* functions or not. This was just one approach and I picked it because I felt like it. I'll change it to passing around lua state if you want

Reply:
Ah, I see now. I am fine with the optional